stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/avatar/AvatarStyle.kt

```diff
@@ -9,13 +9,19 @@ import io.getstream.chat.android.ui.R
 import io.getstream.chat.android.ui.utils.extensions.getDimension
 import io.getstream.chat.android.ui.utils.extensions.use
 
-internal class AvatarStyle(context: Context, attrs: AttributeSet?) {
-    val avatarWidth: Int
-    val avatarHeight: Int
-    val avatarBorderWidth: Int
-    val avatarBorderColor: Int
-    val avatarInitialText: TextStyle
-    val onlineIndicatorEnabled: Boolean
+public class AvatarStyle internal constructor(context: Context, attrs: AttributeSet?) {
```

Comment:
Should we consider re-implementing the builder pattern for this style? It's used in `ChannelListViewStyle` where the old UI components expected it to be a `data class` and so provided a method to change the border color by using `avatarView.copy(...)`, which isn't possible with this one. The fields were originally `val`, but I made them `var` with `internal set` for now.